django/applications/catmaid/static/js/widgets/treenode-table.js

```diff
@@ -486,21 +486,27 @@
       this.oTable.column(1).search(this.filter_nodetype).draw();
     }).bind(this));
 
-    // add ability to conditionally search on confidence
-    $.fn.dataTable.ext.search.push(function(settings, data, dataIndex) {
```

Comment:
Since you added this line in a previous commit of this PR (on which I also commented): would you mind squashing fixes like this into the original commit in the future? This is no big deal, but it would make it easier to review, because I go commit-wise through your changes. Thanks!